sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -91,4 +117,45 @@ class SentryFragmentLifecycleCallbacks(
     private fun getFragmentName(fragment: Fragment): String {
         return fragment.javaClass.simpleName
     }
+
+    private fun isRunningSpan(fragment: Fragment): Boolean =
+        fragmentsWithOngoingTransactions.containsKey(fragment)
+
+    private fun startTracing(fragment: Fragment) {
+        if (!isPerformanceEnabled || isRunningSpan(fragment)) {
+            return
+        }
+
+        var transaction: ISpan? = null
+        hub.configureScope {
+            transaction = it.transaction
+        }
+
+        val fragmentName = getFragmentName(fragment)
+        val span = transaction?.startChild(FRAGMENT_LOAD_OP, fragmentName)
+
+        span?.let {
+            fragmentsWithOngoingTransactions[fragment] = it
+        }
+    }
+
+    private fun stopTracing(fragment: Fragment) {
+        if (!isPerformanceEnabled || !isRunningSpan(fragment)) {
+            return
+        }
+
+        val span = fragmentsWithOngoingTransactions[fragment]
+        span?.let {
+            var status = it.status
+            if (status == null) {
+                status = SpanStatus.OK
+            }
+            it.finish(status)
+            fragmentsWithOngoingTransactions.remove(fragment)
+        }
+    }
+
+    companion object {
+        const val FRAGMENT_LOAD_OP = "fragment.load"
```

Comment:
`h`: Shouldn't this be?
```suggestion
        const val FRAGMENT_LOAD_OP = "ui.load"
```

Reply:
the transaction itself is `ui.load`, but this is the `op` of the span only, so it'd be nice to differ otherwise it'd be hard to understand what is the activity and fragment spans

Reply:
But I think the prefix matters to the Sentry UI. Maybe `ui.fragment.load`? I would clarify this with the visibility team.

Reply:
`ui.fragment.load` sounds good to me.